python/lsst/ts/standardscripts/auxtel/daytime_checkout/latiss_checkout.py

```diff
@@ -0,0 +1,152 @@
+        # super().__init__() above. We can also pass in the script domain and
+        # logger to both classes so log messages generated internally are
+        # published to the efd.
+        self.latiss = LATISS(domain=self.domain, log=self.log)
+
+    @classmethod
+    def get_schema(cls):
+        schema_yaml = """
+            $schema: http://json-schema.org/draft-07/schema#
+            $id: https://github.com/lsst-ts/ts_standardscripts/auxtel/latiss_checkout.yaml
+            title: LatissDayTimeCheckout v1
+            description: LATISS Configuration for DayTimeCheckout.
+            type: object
+            additionalProperties: false
+        """
+        return yaml.safe_load(schema_yaml)
+
+    async def configure(self, config):
+        """Configure script.
+
+        Parameters
+        ----------
+        config : `types.SimpleNamespace`
+            Script configuration, as defined by `schema`.
+        """
+
+        self.config = config
+
+    def set_metadata(self, metadata):
+        """Set estimated duration of the script."""
+
+        metadata.duration = 5.0 * 60  # Approximate 5min to completion.
+
+    async def run(self):
+
+        # Bias Verification, start with checkpoint for observer
+        await self.checkpoint("Bias Frame Verification")
+
+        self.latiss.rem.atoods.evt_imageInOODS.flush()
+        await self.latiss.take_bias(nbias=1)
+        ingest_event = await self.latiss.rem.atoods.evt_imageInOODS.next(
+            flush=False, timeout=10
+        )
+        ingest_event_time = check_evt_time(ingest_event)
+        self.log.info(
+            f"The last ingested image was \n \t {ingest_event.obsid} at {ingest_event_time} UT"
+        )
+        assert ingest_event.statusCode == 0, "Ingestion was not successful!"
+
+        # Engineering test frame verification
+        await self.checkpoint("Engineering Frame Verification")
+
+        available_setup = await self.latiss.get_available_instrument_setup()
+        self.log.info(
+            f"\n The available filters are {available_setup[0]} \n and gratings are {available_setup[1]} "
+        )
+
+        self.latiss.rem.atoods.evt_imageInOODS.flush()
+        await self.latiss.take_engtest(2, filter=0, grating=0)
+        ingest_event = await self.latiss.rem.atoods.evt_imageInOODS.next(
+            flush=False, timeout=10
+        )
+        ingest_event_time = check_evt_time(ingest_event)
+        inst_setup = await self.latiss.get_setup()
+        self.log.info(
+            f"The last ingested image was \n \t {ingest_event.obsid} at {ingest_event_time.utc}"
+            f"UT \n \t with {inst_setup[0]} filter and {inst_setup[1]} grating "
+        )
+        assert ingest_event.statusCode == 0, "Ingestion was not successful!"
+
+    async def assert_feasibility(self):
+        """Verify that the system is in a feasible state to execute the
+        script.
+        """
+        await asyncio.gather(
```

Comment:
I don't thing you need that `gather` here, since there's only one coroutine. Just do

```
await self.latiss.assert_all_enabled()
```

Reply:
Change adopted and applied to other daytime checkout scripts.